test/Microsoft.Azure.Devices.Edge.Test.Common/IPlatform.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public interface IPlatform
+    {
+        Task<string> CollectDaemonLogsAsync(DateTime testStartTime, string filePrefix, CancellationToken token);
+
+        IEdgeDaemon CreateEdgeDaemon(Option<string> installerPath);
```

Comment:
what is difference between installerPath and packagesPath in EdgeDaemon?  Can we combine since both are somehow relative to artifact folder?

Reply:
`installerPath` is the path to `IotEdgeSecurityDaemon.ps1`--it's Windows only because we don't have an installer script on Linux. `packagesPath` points to the DEB files on Linux, CAB file on Windows. Both related to install, but two different things.